### DISCUSSION TITLE
Making Tutorial - User Guides more user friendly (ENGLISH)

### DISCUSSION THREAD

**Discussion by physicsmagician** [fastapi/fastapi#11636] [Questions] [answered]
Hello, all!

As someone with previous Technical Writing experience, I noticed that FastAPI's documentation could use some work. While there is an abundance of useful information on there, as a beginner, I often feel frustrated when trying to understand what I'm reading. Editing the user guides to be more clear and concise will help:

- Reduce time for users to learn how to do something,
- Motivate first-time users to not give up on FastAPI, and
- Creates a more professional look (helps build trust).

While this issue is somewhat similar to Issues in the **English docs #10315** raised by @nilslindemann, this issue pertains specifically to the user guides. However, the suggestions they have included in their discussion are also great and worth looking into.

I'm interested in working on this project (and hopefully aim to help improve the other docs as well). Please let me know what you guys think. :)

**Comment by n8sty** [comment]
I'm not a maintainer but I do lurk around here from time-to-time. My advice is, open a pull request or two that fix one and only one issue with the documentation that you've identified. If that's accepted, then you'll have a decent indication that your approach is useful, otherwise hopefully it's not something that you've sunk too much effort into.

**Reply by physicsmagician** [reply]
Thanks for your suggestion! I'll be sure to try that out.

**Comment by tiangolo** [comment, marked as answer]
Thanks for the interest! I prefer to keep the docs as they are, they have been one of the main "features" people like about FastAPI and they have my intention and style to explain things, so I'll probably prefer to keep the current version. But thanks for the interest! :coffee:
